content/en/docs/kubernetes/collector/components.md

```diff
@@ -661,3 +663,90 @@ subjects:
     name: otel-collector-opentelemetry-collector
     namespace: default
 ```
+
+## Host Metrics Receiver
+
+| Deployment Pattern   | Usable                                                        |
+| -------------------- | ------------------------------------------------------------- |
+| DaemonSet (Agent)    | Preferred                                                     |
+| Deployment (Gateway) | but will only collect metrics from the node it is deployed on |
+| Sidecar              | No                                                            |
+
+The Host Metrics Receiver collects metrics from a host using a variety of
+scrappers. There is some overlap with the
+[Kubeletstats Receiver](#kubeletstats-receiver) so if you decide to use both it
+may be worth it to disable these duplicate metrics.
+
+In Kubernetes the receiver needs access to the `hostfs` volume to work properly.
+If you're using the [OpenTelemetry Collector Helm chart](../../helm/collector/)
+you can use the
+[`hostMetrics` preset](../../helm/collector/#host-metrics-preset) to get
+started.
+
+The available scrapers are:
+
+| Scraper      | Supported OSs                | Description                                            |
+| ------------ | ---------------------------- | ------------------------------------------------------ |
+| [cpu]        | All except Mac<sup>[1]</sup> | CPU utilization metrics                                |
+| [disk]       | All except Mac<sup>[1]</sup> | Disk I/O metrics                                       |
```

Comment:
Why not just say `Linux, Windows` like further below, where it says `Linux, Mac`?

If you'd like to proceed with using `All except Mac<sup>[1]</sup>`, then there's also a missing footnote for the `[1]`.

Reply:
Also, you can use markdown footnote syntax `^[1]`.